src/app/project/project.component.scss

```diff
@@ -30,6 +30,23 @@
       margin: 0;
       width: 111%;
       max-width: 100%;
+
+      .project-logos {
+        padding: 0;
+        list-style-type: none;
+        display: flex;
+
+        li {
+          margin-right: 1rem;
+
+          a {
+            img {
+              max-height: 90px;
```

Comment:
Have you verified this works as expected in Chrome, Firefox and Safari? 

I am not sure if it has changed, but previously there was an issue using just max-height without defining a height in one of the major non-IE browsers (I think it was FF but don't recall) - the height would get returned as 0px.